controllers/cartController.js

```diff
@@ -0,0 +1,53 @@
+const cartService = require("../services/cartService");
+
+const cartIn = async (req, res) => {
+  try {
+    const { userId, productId, quantity } = req.body;
+    if (!userId || !productId || !quantity) {
+      const err = new Error("Invalid user's Id or quantity");
+      err.statusCode = 400;
+      throw err;
+    }
+    const cart = await cartService.cartIn(userId, productId, quantity);
```

Comment:
cart라는 변수를 사용하지 않기 때문에 장바구니를 새로 생성하는 service 함수의 결과를 변수에 할당하지 않아도 될 것 같습니다.
```suggestion
    await cartService.cartIn(userId, productId, quantity);
```